src/modules/function/declaration_utils.rs

```diff
@@ -53,6 +53,9 @@ pub fn is_functions_comment_doc(meta: &mut ParserMetadata) -> bool {
                 return false;
             }
         }
+        if tok.word.starts_with("#[") {
```

Comment:
but this will allow to ignore comments like the function comments in stdlib?

Reply:
The fact that function comments weren't detected was because in `src/math` we use preprocessor directive (`#[...]`) that doesn't get detected by the algorithm.

Reply:
but the issue there was also in other stdlib stuff not just in math

Reply:
When I checked this - it was everywhere were the preprocessor directive was used

Reply:
After testing the changes on this branch, does it resolve the issues you observed?

Reply:
I didn't checked if works but i saw that the comments were kept also on functions that doesn't have those attributes.